src/yb/tserver/pg_client.proto

```diff
@@ -275,7 +275,7 @@ message PgGetTServerUUIDRequestPB {
 
 message PgGetTServerUUIDResponsePB {
   AppStatusPB status = 1;
-  string node_uuid = 2;
+  repeated uint64 top_level_node_id = 2;
```

Comment:
How many times can this be repeated?

Consider using something like
`repeated bar arr    = 1 [length=10];`

Reply:
There doesn't seem to be any such way to set the length of a repeated field